optaplanner-examples/src/main/java/org/optaplanner/examples/common/app/CommonApp.java

```diff
@@ -156,7 +157,7 @@ public void init(Component centerForComponent, boolean exitOnClose) {
         return new AbstractSolutionImporter[0];
     }
 
-    protected AbstractSolutionExporter createSolutionExporter() {
+    protected Set<AbstractSolutionExporter> createSolutionExporters() {
         return null;
```

Comment:
`Collections.emptySet()` works better in this situation, I think.

Reply:
I agree.